.github/workflows/build-yocto.yml

```diff
@@ -63,10 +63,9 @@ jobs:
 
       - name: Kas build
         run: |
-          export DL_DIR=/srv/gh-runners/quic-yocto/downloads
-          export SSTATE_DIR=/srv/gh-runners/quic-yocto/sstate-cache
+          export PERSISTENT_DIR=/srv/gh-runners/quic-yocto
+          export DL_DIR=${PERSISTENT_DIR}/downloads
+          export SSTATE_DIR=${PERSISTENT_DIR}/sstate-cache
           mkdir -p $DL_DIR
           mkdir -p $SSTATE_DIR
-          mkdir build
-          cd build
-          kas build ../ci/${{ matrix.machine }}.yml
+          kas build ci/${{ matrix.machine }}.yml
```

Comment:
I'm not a fan of of building as part of the main git repo, maybe remove but set KAS_BUILD_DIR to a folder outside the repo?

Reply:
done